manim/utils/config_utils.py

```diff
@@ -397,3 +433,7 @@ def _run_config():
     # this is for internal use when writing output files
     file_writer_config = _parse_file_writer_config(config_parser, args)
     return args, config_parser, file_writer_config, successfully_read_files
+
+def curr_config_dict():
+    config=_run_config()[1]
+    return {section: dict(config[section]) for section in config.sections()}
```

Comment:
Question: this is not _really_ the actual config dictionary, is it? This is just the result of reading the cascading config files. I would call this the finalized config parser. The actual finalized global config dictionary can only be obtained after _parse_config has been called from `config.py`.

Reply:
Good point. I've renamed it to `finalized_configs_dict()` in the latest commit, since it doesn't return an ArgumentParser, but it's contents in dictionary form.